src/tests/3-corax+.8o

```diff
@@ -111,13 +121,32 @@
   x2 := 27
   y := 1
   drawop im2 imX
+	
+	v0 := 0
+	# call subroutine, which sets v0 to 1, and displays a checkmark
   test2X
+	
+	# 2NNN fail, skips the 00EE test entirely as to not crash the interpreter.
+	if v0 == 0 then jump skip0E
 
-  #test 0E
-  y += 5
-  drawop im0 imE
+  #test 0E pass
   i := image-ok
   sprite x2 y 4
+	jump after0E
+	
+	# interpreter ends up here when 2NNN isn't implemented.
+: skip0E
+	i := image-no
+	sprite x2 y 4
+	y += 5
+	drawop im0 imE
```

Comment:
I'd rather show nothing for `00EE` instead of a cross when `2NNN` isn't implemented, as technically it may work fine -- we just haven't tested it.